components/Calculator.tsx

```diff
@@ -0,0 +1,226 @@
+import {
+  toWei,
+  useGetGGPPriceInAVAX,
+  useGetRewardsEligibilityMinSeconds,
+  weiValue,
+} from "@/hooks/mounted";
+import { useStakers } from "@/hooks/mounted";
+import { Button, Col, Input, Row, Space, Table, Typography } from "antd";
+import { BigNumber } from "ethers";
+import { commify, formatEther } from "ethers/lib/utils.js";
+import { useEffect, useState } from "react";
+
+const { Title } = Typography;
+
+export function Calculator() {
+  const [ggpAmount, setGGPAmount] = useState<number>(0);
+  const [avaxAmount, setAvaxAmount] = useState<BigNumber>(BigNumber.from(1000));
+
+  const nodeOpPot = BigNumber.from("45749504487698707175523");
+  const investorPot = BigNumber.from("5083278276410967463947");
+
+  const { data: stakers } = useStakers();
+  const { data: minSeconds } = useGetRewardsEligibilityMinSeconds();
+  const { data: ggpPriceInAVAX } = useGetGGPPriceInAVAX();
+
+  if (!stakers || !minSeconds || !ggpPriceInAVAX) return <Space></Space>;
```

Comment:
```suggestion
  if (!stakers || !minSeconds || !ggpPriceInAVAX) return null;
```

Space is for putting spaces between elements